tests/integration/04.Edge.Tests.ps1

```diff
@@ -317,6 +317,31 @@ Describe "Edge" {
             $rtg.bgp.enabled | should be "false"
         }
     }
+    
+    Context "HighAvailability" {
+      it "Can enable HA" {
+        $esg = Get-NsxEdge $name
+        $esg | should not be $null
+        $esg.features.highAvailability.enabled = "true"
+        $esg | Set-NsxEdge -confirm:$false
+        $esg = Get-NsxEdge $name
+        $esg.features.highAvailability.enabled | should be "true"
+      }
+      it "Can failover between HA pair" {
+        Write-Host " getting nsx edge" 
+        $esg = Get-NsxEdge $name
+        $esg | should not be $null
+        # test the current HA status
+        $esg.features.highAvailability.enabled | should be "true"
```

Comment:
Shouldn't this check HA first, and if not enabled, then enabled HA. It assumes that the previous test has completed successfully.